### PR TITLE
Formatting doc

### PR DESCRIPTION
**Pull request by acucunato**
fixed api bug, added catch function to ajax calls

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -130,58 +136,68 @@ $(document).ready(function() {
...
       method: "GET"
-    }).then(function(response) {
-      if (response.status === 400) {
-        return $(".breedInfo").text(
-          "No hounds by the name of '" +
-            breed +
-            "' exist in this land! Double check the letters on your scroll!"
-        );
-      }
+    })
```
Comment (acucunato) [inline script.js:139]: all of the changes below were just spacing.. i used prettier to format so i didnt change any of jordans code

File: script.js [hunk 2]
```diff
@@ -130,58 +136,68 @@ $(document).ready(function() {
...
-      bredFor.appendTo(infoCard);
+        var bredFor = $("<p>");
+        bredFor.text("Bred For: " + response[0].bred_for);
+        bredFor.appendTo(infoCard);
 
-      infoCard.appendTo("#breedSection");
-    });
+        infoCard.appendTo("#breedSection");
+      })
+      .catch(function() {
```
Comment (acucunato) [inline script.js:194]: added this to catch an api error

File: index.html [hunk 3]
```diff
@@ -9,7 +9,6 @@
   <link rel="stylesheet" href="style.css">
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
   <link href="https://fonts.googleapis.com/css?family=Lato|MedievalSharp|Berkshire+Swash&display=swap" rel="stylesheet">
-  <link href="https://fonts.googleapis.com/css?family=&display=swap" rel="stylesheet">
```
Comment (acucunato) [inline index.html:12]: wasnt linking to anything.. above it is our google fonts link

File: script.js [hunk 4]
```diff
@@ -37,7 +37,7 @@ $(document).ready(function() {
             return response.json();
           })
           .then(function(response) {
-            if (response.status === 400) {
+            if (response.status === 400 || response.animals.length === 0) {
```
Comment (acucunato) [inline script.js:40]: ajax call fix

**Review by jsmithxyz — APPROVED** [review]

**Pull request merged**